sql/understanding-the-query-execution-plan.md

```diff
@@ -30,8 +30,32 @@ TiDB 优化器会根据当前数据表的实际情况来选择最优的执行计
 | operator info | 每个 operator 的详细信息。各个 operator 的 operator info 各有不同，我们将在 [Operator Info](#operator-info) 中详细介绍                   |
 | count         | 预计当前 operator 将会输出的数据条数，基于统计信息以及 operator 的执行逻辑估算而来                                                            |
 
+## <span id="introduction">概述</span>
```

Comment:
You don't need to specify a destination anchor manually because we can use the built-in function in Markdown for Table of Content. For example, this section can be represented as simple as 概述。